yarn.lock

```diff
@@ -6447,9 +6447,9 @@ isurl@^1.0.0-alpha5:
     has-to-string-tag-x "^1.2.0"
     is-object "^1.0.1"
 
-"jellyfin-noto@https://github.com/jellyfin/jellyfin-noto":
-  version "1.0.2"
-  resolved "https://github.com/jellyfin/jellyfin-noto#a441b179c833288fc372cadb408d32a76c5479f1"
+"jellyfin-noto@https://github.com/ferferga/jellyfin-noto":
+  version "1.0.3"
+  resolved "https://github.com/ferferga/jellyfin-noto#3f4b90ccaff4fcbe2d68557a2b6d5f40028df8e8"
```

Comment:
Why is your fork here?

Reply:
I thought I discarded that change, sorry 🤦‍♂️